app/helpers/link_to_helper.rb

```diff
@@ -254,6 +254,20 @@ def unhappy_url(info_request = nil)
     end
   end
 
+  # Google Analytics
+
+  # if used, value must an integer
+  def analytics_event_code(category, action, label=nil, value=nil, label_is_script=false)
+    value = 1 unless value
```

Comment:
I'd probably do something like:

begin
  value = Integer(value)
rescue ArgumentError
  raise ArgumentError, %Q(:value option must be an Integer: "#{ options[:value] }")
end